app/src/main/AndroidManifest.xml

```diff
@@ -1,153 +1,150 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
-          package="org.mifos.mobile">
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
I see, there are so many changes that is not relevant to this PR. Please remove them.